### ISSUE TITLE
The overriding of TargetPlatformIdentifier to UAP in Microsoft.ApplicationModel.Resources.PriGen.targets should not be specific to .NET5, it should accommodate newer versions. 

### ISSUE THREAD

**Issue by rohanp-msft** [microsoft/ProjectReunion#596]
How to repro:
Use WinUI3 from Project Reunion with .NET MAUI. .NET MAUI depends on NET6.

**Comment by rohanp-msft** [comment]
Fixed in #707

**Comment by rohanp-msft** [comment]
The implementation of this is incorrect per Daniel Plaisted's comment in https://github.com/microsoft/ProjectReunion/pull/707. Will fix this.

**Comment by rohanp-msft** [comment]
On second thought, I'll track the impl mod with a new issue: https://github.com/microsoft/ProjectReunion/issues/716
